tests/generate_plan_tests.rs

```diff
@@ -93,6 +93,23 @@ fn test_yarn() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_yarn_berry() -> Result<()> {
```

Comment:
Could you please add a docker run test for this too. Also for the PNPM stuff would also be useful. Should just a few lines for each test.

Reply:
Oh yeah, I forgot about that.